ecosystem/python/sdk/aptos_sdk/aptos_token_client.py

```diff
@@ -515,6 +517,12 @@ async def mint_soul_bound_token(
         )
         return await self.client.submit_bcs_transaction(signed_transaction)
 
+    # :!:>transfer_token
+    async def transfer_token(
+        self, owner: Account, token: AccountAddress, to: AccountAddress
+    ) -> str:
+        return await self.client.transfer_object(owner, token, to) # <:!:transfer_token
+
```

Comment:
As a minimalist, I'm inclined to oppose this. It masks from the user a real operation and instead gives a pseudo-operation. but ahh... sigh


also why are we showing the body of the function?

it feels inconsistent with the mint? if anything is really actionable, it would be that...

Reply:
Hmm, I mean I'm okay with removing it. I just figured it might be nice to have an obvious transfer function in the AptosTokenClient itself. But I see your point, it is most likely unnecessary, especially since this is going in the tutorial.

I'll remove the doc annotation from this and annotate the transfer_object function declaration then just explain how to use it in the tutorial

Reply:
I don't really care all that much, just major issue is we expose the return statement here but not in the mint function.

We won't be changing the SDK all that much in this space, so if it is convenient who am I to argue...

Reply:
Yeah I see what you're saying, I only included it so that it's obvious it's just calling the `transfer_object` function under the hood. But I can just explain that